extensions/standard-processors/tests/unit/TailFileTests.cpp

```diff
@@ -1521,8 +1521,8 @@ TEST_CASE("TailFile interprets the lookup frequency property correctly", "[multi
     REQUIRE(LogTestController::getInstance().contains("Logged 2 flow files"));
   }
 
-  SECTION("Lookup frequency set to 10 ms => new files are only picked up after 10 ms") {
-    plan->setProperty(tail_file, processors::TailFile::LookupFrequency.getName(), "10 ms");
+  SECTION("Lookup frequency set to 100 ms => new files are only picked up after 100 ms") {
```

Comment:
Is this change meant to be in this PR?

Reply:
yes, otherwise this test fails on windows most of the time

Reply:
thanks for fixing this